paymentsheet/src/test/java/com/stripe/android/paymentsheet/forms/FormViewModelTest.kt

```diff
@@ -59,6 +59,21 @@ internal class FormViewModelTest {
 
     private val showCheckboxFlow = MutableStateFlow(false)
 
+    @Test
+    fun `Verify completeFormValues is not null when no elements exist`() = runTest {
```

Comment:
This tests the new behavior.

Reply:
these comments on this PR were helpful. Thank you!